src/Tempest/Router/tests/Responses/BackResponseTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Router\Tests\Responses;
+
+use Tempest\Http\Method;
+use Tempest\Http\Status;
+use Tempest\Router\GenericRequest;
+use Tempest\Router\Header;
+use Tempest\Router\Request;
+use Tempest\Router\Responses\Back;
+use Tests\Tempest\Integration\FrameworkIntegrationTestCase;
+
+/**
+ * @internal
+ */
+final class BackResponseTest extends FrameworkIntegrationTestCase
```

Comment:
You should move this test to the integration test folder instead: https://github.com/tempestphp/tempest-framework/tree/main/tests/Integration